docs/proxy/datacenter_proxy/examples.md

```diff
@@ -40,15 +38,15 @@ const Apify = require("apify");
 
 Apify.main(async () => {
     const requestList = await Apify.openRequestList(
-        "my-list", ["http://www.example.com"]
+        "my-list", ["http://proxy.apify.com/?format=json"]
     );
     const proxyConfiguration = await Apify.createProxyConfiguration();
 
     const crawler = new Apify.PuppeteerCrawler({
         requestList,
         proxyConfiguration,
         handlePageFunction: async ({ page, request, proxyInfo }) => {
-            return Apify.pushData({ title: await page.title() });
+            console.log(page);
```

Comment:
`console.log(page)` won't do anything good because the `page` is not nicely loggable. `console.log(await page.content())` would be better.

I'm sorry if I made it look like you need to remove all complexity from the examples. I meant only the unnecessary one, like figuring out selectors, or setting up `gotoFunction` and blocking requests and so on.